### PR TITLE
fix(material-ui): fix RangeWidget onChange handler #2161

### PR DESCRIPTION
**Pull request by JonCatmull**
fix: #2161 

RangeWidget's onChange handler for material-ui package was broken due to conditional being the wrong way around. This caused new non empty values to be changed to options.emptyValue on change.

### CODE REVIEW

**Review by JonCatmull — COMMENTED** [review]
RangeWidget's onChange handler for material-ui package was broken due to conditional being the wrong way around. This caused new non empty values to be changed to `options.emptyValue` on change.

File: packages/material-ui/src/RangeWidget/RangeWidget.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const RangeWidget = ({
   const sliderProps = { value, label, id, name: id, ...rangeSpec(schema) };
 
   const _onChange = (_: any, value?: number | number[]) => {
-    onChange(value ? options.emptyValue : value);
+    onChange(value ? value : options.emptyValue);
```
Comment (heath-freenome) [inline packages/material-ui/src/RangeWidget/RangeWidget.tsx:22]: It it likely that the same code is in `packages/mui/src/RangeWidget/RangeWidget.tsx`. Please update that as well

Comment (JonCatmull) [reply packages/material-ui/src/RangeWidget/RangeWidget.tsx:22]: Good point, done.

**Comment by heath-freenome** [comment]
@JonCatmull Oh shoot, I forgot to ask you to upgrade the `CHANGELOG.md` file to describe these fixes for both `@rjsf/material-ui` and `@rjsf/mui`

**Comment by heath-freenome** [comment]
> @JonCatmull Oh shoot, I forgot to ask you to upgrade the `CHANGELOG.md` file to describe these fixes for both `@rjsf/material-ui` and `@rjsf/mui`

NVM, I'm making a fix and will add them myself so that we can get this merged quicker

**Review by heath-freenome — APPROVED** [review]

**Pull request merged**